### PR TITLE
docs: add contributing markdown files

### LINKED ISSUE #13: Contributor docs
Contributor documents are needed, such as code of conduct and contributing agreement. A security document (SECURITY.MD) is also needed.

### PR DESCRIPTION
**Pull request by Nicholaiii**
This closes #13.
Please note that your email is not public, so I couldn't add it to the `SECURITY.md`, you will have to change it to your email address, or ask me to remove the file, if you don't like it.

### CODE REVIEW

**Comment by warriordog** [comment]
@Nicholaiii 
Thanks for the PR! This all looks good, but could you add "acomputerdog@gmail.com" for the contact email?

BTW thanks for letting me know about my email, I forgot that it was still set to private.

**Comment by Nicholaiii** [comment]
Consider it done! :)
You can also consider adding the more verbose version of the `author` field in your `package.json`:
```json
{
  "author": {
    "name": "Nicholai Nissen",
    "email": "nicholainissen@gmail.com",
    "url": "https://nicholai.dev"
  }
}
```

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (warriordog) [inline CODE_OF_CONDUCT.md:64]:
```suggestion
acomputerdog@gmail.com.
```
Would you mind inserting my email here too?

**Review by warriordog — CHANGES_REQUESTED** [review]
This all looks good, but there is a placeholder left in the CODE_OF_CONDUCT.md file. Would you mind filling that in?

**Comment by warriordog** [comment]
Thanks for the info about the long-form `author` field, I wasn't aware that you could do that.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Nicholaiii) [reply CODE_OF_CONDUCT.md]: Done!

**Review by warriordog — APPROVED** [review]
Looks good, thanks for your contribution!

**Pull request merged**